src/endpoints/pcm.js

```diff
@@ -0,0 +1,13 @@
+import CRUDExtend from '../extends/crud'
+
+class PCMEndpoint extends CRUDExtend {
+  constructor(endpoint) {
+    const config = { ...endpoint } // Need to clone config so it is only updated in PCM
```

Comment:
We need to make a copy of the object, so when we update, it does not update the version in the other services (this has been tested, and needs to be done.)